### PR TITLE
add-aws-ipi-ovn-migration-sdn-workflow

### PR DESCRIPTION
**Pull request by weliang1**
ovn-migration-sdn-workflow will install OVN cluster and execute cucushift-upgrade-prepare, ovn-sdn-migration-rollback script, and cucushift-upgrade-check. 

Comparing the testing results  between [vsphere cluster](https://github.com/openshift/release/pull/37660) and this aws cluster. Vsphere cluster always fail on different steps/cases either in  cucushift-upgrade-prepare or cucushift-upgrade-check, test results did not show any errors when execute  ovn-sdn-migration-rollback script. Seem vsphere cluster is not stable for ovn-migration-sdn-workflow testing, let’s review and merge this ovn-migration-sdn-workflow for aws ipi cluster. I will close https://github.com/openshift/release/pull/37660 after this PR merged.

As I mentioned before, comparing sdn2ovn migration, we do not think many customers will do the ovn2sdn migration, so one workflow testing should be enough.

@liangxia @zhaozhanqi PTAL

### CODE REVIEW

**Comment by weliang1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-migration-sdn-p2-f7

**Comment by weliang1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-migration-sdn-p2-f7

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1049,6 +1049,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-kubevirt-hypershift
+- as: aws-ipi-ovn-migration-sdn-p2-f7
+  cron: 58 4 6,14,22,29 * *
+  steps:
+    cluster_profile: aws
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1055]: Shall we use `aws-qe` ?

Comment (weliang1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1055]: @liangxia Good catch! I will update it. 

I always have the problems when I update the code and re commit in release repo, could you help to check if below steps are correct, thanks!
```
sudo make registry-metadata
sudo make ci-operator-config
sudo make jobs
git add openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml
git add openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml
git commit –amend
git push --force-with-lease origin $branchname
```

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1055]: Try this,
```
git fetch upstream master
git rebase upstream/master
# change codes
sudo make update
git add xxx yyy ... zzz
git commit  --amend
git push origin $branchname --force
```

**Pull request closed without merging**